### PR TITLE
fix: (CXSPA-2243) - Improved a11y for popovers in 'organization'

### PR DESCRIPTION
**Pull request by Pio-Bar**
https://jira.tools.sap/browse/CXSPA-2243

Improved screen reader readout for information buttons in 'My Company' subpages.

### CODE REVIEW

**Review by rmch91 — DISMISSED** [review]

**Review by developpeurweb — CHANGES_REQUESTED** [review]
This works better now for both JAWS and VO.
There are three ways of closing the popover while using the keyboard, regardless of Screen Reader use, either pressing Spacebar or Enter will toggle the display and the narration, and then continue tabbing to the next element without breaking the order.
However, a third way is pressing the ESC key, and this resets the focus to the top of the page thus breaking the order.
Is there a way to set the ESC key to only close the popover without the focus moving to the top of the page and continue with the natural order expectation?

**Comment by Pio-Bar** [comment]
I've modified it to restore focus to the opener element regardless of how it was closed.

File: projects/storefrontlib/shared/components/popover/popover.service.ts [hunk 1]
```diff
@@ -31,13 +31,9 @@ export class PopoverService {
     return config;
   }
 
-  setFocusOnElement(
-    element: ElementRef,
-    focusConfig: FocusConfig,
-    appendToBody?: boolean
-  ) {
-    if (focusConfig && appendToBody) {
+  setFocusOnElement(element: ElementRef) {
```
Comment (Zeyber) [inline projects/storefrontlib/shared/components/popover/popover.service.ts]: Breaking changes need handling :)

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: projects/storefrontlib/shared/components/popover/popover.service.ts (hunk 1 above)
Comment (Pio-Bar) [reply projects/storefrontlib/shared/components/popover/popover.service.ts]: Feature flags added ;)

**Review by Zeyber — DISMISSED** [review]

**Review by developpeurweb — DISMISSED** [review]

**Review by Zeyber — APPROVED** [review]

**Pull request merged**